src/js/game/core.js

```diff
@@ -425,7 +425,12 @@ export class GameCore {
 
         root.camera.transform(context);
 
-        assert(context.globalAlpha === 1.0, "Global alpha not 1 on frame start");
+        if (context.globalAlpha !== 1.0) {
+            console.warn("Global Alpha not set back to 1 on Frame Begin");
+            context.globalAlpha = 1;
+        }
+
+        // assert(context.globalAlpha === 1.0, "Global alpha not 1 on frame start");

```

Comment:
I'm going to go out on a limb and assume that this is against what you want. Personally, I found that it kept happening and I couldn't track down where the issue was originating. Since changing it over to the if statement I haven't had any issues and I feel like it's a better solution overall. Notifying the developer of the problem but fixing it so it doesn't have effects.